packages/react/src/components/ThemeProvider/ThemeContext.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import { defaultTheme, WebTheme } from '@aws-amplify/ui';
+
+export type ColorMode = 'system' | 'light' | 'dark';
+
+export interface ThemeContextType {
+  theme: WebTheme;
+  colorMode?: ColorMode;
+  setColorMode?: React.Dispatch<React.SetStateAction<ColorMode>>;
+  root?: boolean;
```

Comment:
Should we use our boolean standard here and prefix with `is`?
```suggestion
  isRootTheme?: boolean;
```